cmd/oam-kubernetes-runtime/main.go

```diff
@@ -45,6 +45,8 @@ func main() {
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&leaderElectionNamespace, "leader-election-namespace", "default",
```

Comment:
```suggestion
	flag.StringVar(&leaderElectionNamespace, "leader-election-namespace", "",
```

The default value should be empty